bin/auth-api/src/routes/auth_token.routes.ts

```diff
@@ -75,38 +86,41 @@ router.get("/workspaces/:workspaceId/authTokens/:authTokenId", async (ctx) => {
   ctx.body = { authToken };
 });
 
-// rename the given authToken for the given workspace
-router.put("/workspaces/:workspaceId/authTokens/:authTokenId", async (ctx) => {
-  const { authToken } = await authorizeAuthTokenRoute(ctx, [RoleType.OWNER]);
-
-  // Get params from body
-  const { name } = validate(
-    ctx.request.body,
-    z.object({
-      name: z.nullable(z.string()),
-    }),
-  );
-
-  await updateAuthToken(authToken.id, { name });
-
-  ctx.body = { authToken };
-});
-
 // revoke the given authToken for the given workspace
 router.post("/workspaces/:workspaceId/authTokens/:authTokenId/revoke", async (ctx) => {
-  const { authToken } = await authorizeAuthTokenRoute(ctx, [RoleType.OWNER]);
+  const { authToken, authUser, workspace } = await authorizeAuthTokenRoute(ctx, [RoleType.OWNER]);
 
   await updateAuthToken(authToken.id, { revokedAt: new Date() });
 
+  tracker.trackEvent(authUser, "workspace_api_token_revoked", {
+    workspaceId: workspace.id,
+    workspaceName: workspace.displayName,
+    tokenName: authToken.name,
+    tokenCreated: authToken.createdAt,
+    tokenRevoked: new Date(),
```

Comment:
We probably also want to report `tokenExpires` for consistency (and because I can imagine wanting to know how close to the expiration date people revoke tokens).

This also makes me realize the endpoint has a bug--if we're going to return an authToken it should be the *updated* one with `authToken.revokedAt`. If it had the new date, you would have been able to use a common method to construct the event properties across these endpoints. (With the exception of tokenDeleted). (It is obviously not necessary to fix this bug in your PR, I'm just writing to remind myself.)